_data/eohomegrownapps/eohomegrownapps.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Euan Ong
+institution: Magdalen College School, Oxford
+profile_pic: EuanOng.jpg
+quote: we conjure the spirits of the computer with our spells (SICP) -- λx.λy.x
```

Comment:
Hello @eohomegrownapps  ,

I'm reviewing your application to help speeding the process before the actual github employees do it.
You should not have special characters in this file. It says in the readme `Do not use special characters in the template above.`
Please make the needed changes as soon as possible.

Reply:
I mean the character `λ`

Reply:
Apologies for this -- done :)

Reply:
https://github.com/education/graduation/issues/348 seems to suggest the lambda character λ might be allowed?

Reply:
Apparently the printable ascii are just from the range decimal 32 to 127

Reply:
ah -- I was somewhat confused about this, but "We can do emojis as well" seems to suggest they might have meant printable unicode? (also characters with accents aren't in printable ASCII)